core/src/win32/filed/vss.cc

```diff
@@ -223,44 +210,38 @@ bool VSSClient::GetShadowPathW(const wchar_t* szFilePath,
 
 const size_t VSSClient::GetWriterCount()
 {
-  return pAlistWriterInfoText_->size();
+  return writer_info_.size();
 }
 
-const char* VSSClient::GetWriterInfo(int nIndex)
+const char* VSSClient::GetWriterInfo(size_t nIndex)
 {
-  return (char*)pAlistWriterInfoText_->get(nIndex);
+  if ( nIndex < writer_info_.size() ) {
+    return writer_info_.at(nIndex).info_text_.c_str();
```

Comment:
you just did the bounds check the line before, so you can use [] instead of at().
```suggestion
    return writer_info_[nIndex].info_text_.c_str();
```